packages/types/src/rule/code/E1004.ts

```diff
@@ -10,6 +10,6 @@ export const message: RuleMessage = {
   description: `
 #### Description
 
-Detect if there is a low version of ECMA syntax in the js bundle file
+Detect if there is a bigger version of ECMA syntax in the js bundle file
```

Comment:
`newer` is better for describing version.

```suggestion
Detect if there is a newer version of ECMA syntax in the js bundle file
```